.github/workflows/test-helm-chart.yaml

```diff
@@ -21,22 +21,22 @@ jobs:
         # specified below, these parameters have no meaning on their own and
         # gain meaning on how job steps use them.
         include:
+          - k3s-channel: v1.23
+            test: install
           - k3s-channel: v1.22
             test: install
           - k3s-channel: v1.21
             test: install
           - k3s-channel: v1.20
             test: install
-          - k3s-channel: v1.19
-            test: install
 
           # We run an upgrade test where we first install an already released
           # Helm chart version and then upgrade to the version we are now
           # testing. We test upgrading from the latest main version.
           #
           # It can be very useful to see the "Helm diff" step's output from the
           # latest dev version.
-          - k3s-channel: v1.19
+          - k3s-channel: v1.21
```

Comment:
What's the reason for using v1.21 here?

Reply:
We are currently running the upgrade only on a single platform (with this change 1.21). I've picked that because it's one of the versions most used by our users.